spec.js

```diff
@@ -23,17 +25,50 @@ describe('Choko - Create account', () => {
 });
 
 describe('Choko - Blog', () => {
-  it('try to create blog post without permission', () => {
-    const ramdonName = faker.lorem.word();
-    const randomTitle = faker.lorem.words();
-    const randomText = faker.lorem.sentences();
+  // instantiating a new instance of CreateBlogPost
```

Comment:
I think this comment is not necessary. Reading the code is code enough for understanding what it is doing.